docs/source/core_services/drivers/driver_configuration/ecobee_web_driver.rst

```diff
@@ -141,69 +148,110 @@ This configuration works as follows:
 For additional explanation on the quirks of Ecobee's readable/writable points, visit:
 https://www.ecobee.com/home/developer/api/documentation/v1/functions/SetHold.shtml
 
+
 Installation
 ############
 
-These are the most basic installation steps for the Ecobee driver. This guide
-assumes the user is in the VOLTTRON_ROOT directory, the VOLTTRON platform has
-been installed and bootstrapped per the  instructions in the VOLTTRON README.
+These are the most basic installation steps for the Ecobee driver.
 
-Below are the setup instructions.
+.. note::
 
-::
+    This guide assumes the user is in the VOLTTRON_ROOT directory, the VOLTTRON platform has been installed and
+    bootstrapped per the instructions in the VOLTTRON README.
+
+Below are the setup instructions for the Ecobee driver.
+
+
+1. If the platform has not been started:
+
+.. code-block:: Bash
+
+    ./start-volttron
+
+2. Be sure that the environment has been activated - you should see (volttron) next to <user>@<host> in your terminal
+window. To activate an environment, use the following command.
 
-    1. If the platform has not been started:
+.. code-block:: Bash
 
-        ./start-volttron
+    . env/bin/activate
 
-    2. If the environment has not been activated - you should see (volttron) next to <user>@<host> in your terminal window
+3. Install a Master Driver if one is not yet installed
 
-        . env/bin/activate
+.. code-block:: Bash
 
-    4. Install a Master Driver if one is not yet installed
+    python scripts/install-agent.py -s services/core/MasterDriverAgent -c <master driver configuration file>
 
-        python scripts/install-agent.py -s services/core/MasterDriverAgent -c <master driver configuration file>
+4. Load the driver configuration into the configuration store ("vctl config list platform.driver" can be used to show
+   installed configurations)
 
-    5. Load the driver configuration into the configuration store ("vctl config list platform.driver" can be used to show installed configurations)
+.. code-block:: Bash
 
-        vctl config store platform.driver <device topic (mentioned in driver configuration section)> <path to driver configuration>
+    vctl config store platform.driver <device topic (mentioned in driver configuration section)> <path to driver configuration>
```

Comment:
lets go ahead and use specific example setup rather than the <device topic > <path to driver>

i.e. Create a directory called configs create a file in that directory called ecobee.config.json and ecobee.csv

... use that through out the example process.